pkg/core/block/header.go

```diff
@@ -238,6 +238,7 @@ func (b *Header) UnmarshalJSON(data []byte) error {
 }
 
 // GetExpectedHeaderSize returns the expected Header size with the given number of validators.
+// For calculation genesis block Header size numOfValidators should be 0.
```

Comment:
Either "For genesis block Header size calculation" or "To calculate genesis block Header size".